src/style.css

```diff
@@ -1,3 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  background-color: #858080;
+}
+
+html {
+  /* stylelint-disable-next-line string-quotes */
+  font-family: 'Roboto', sans-serif;
+  --color-hover: #00acdc;
+  --color-text: rgb(0, 0, 0);
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
```

Comment:
No need to set max-width, it should be full width on large screens

Reply:
ok